dist/localforage-removeitems.es6.js

```diff
@@ -33,30 +33,31 @@ function removeItemsIndexedDB(keys, callback) {
             var dbInfo = localforageInstance._dbInfo;
             var transaction = dbInfo.db.transaction(dbInfo.storeName, 'readwrite');
             var store = transaction.objectStore(dbInfo.storeName);
+            var lastError;
+
+            transaction.oncomplete = function () {
+                resolve();
+            };
+
+            transaction.onabort = transaction.onerror = function () {
+                reject(lastError || transaction.error || 'Unknown error');
+            };
+
+            function requestOnError(evt) {
+                var request = evt.target || this;
+                lastError = request.error || request.transaction.error;
```

Comment:
How about storing the first error and check whether it is already set before `reject`ing?